azure-pipelines.yml

```diff
@@ -38,181 +37,190 @@ variables:
-              Build_Debug:
-                _BuildConfig: Debug
-                _SignType: test
+resources:
+  repositories:
+  - repository: 1esPipelines
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
 
-        steps:
-        - checkout: self
-          clean: true
-        - ${{ if ne(variables['System.TeamProject'], 'public') }}:
-          - task: PowerShell@2
-            displayName: Setup Private Feeds Credentials
-            inputs:
-              filePath: $(Build.SourcesDirectory)/eng/common/SetupNugetSources.ps1
-              arguments: -ConfigFile $(Build.SourcesDirectory)/NuGet.config -Password $Env:Token
-            env:
-              Token: $(dn-bot-dnceng-artifact-feeds-rw)
-        # Use utility script to run script command dependent on agent OS.
-        - script: eng/common/cibuild.cmd
-            -configuration $(_BuildConfig)
-            -prepareMachine
-            -integrationTest
-            $(_InternalBuildArgs)
-            $(_InternalRuntimeDownloadArgs)
-          displayName: Windows Build / Publish
+# This is necessary because 1ES doesn't work using strategy & matrix
+parameters:
+- name: buildConfigurations
+  type: object
+  default:
+  # Always build the Release configuration, but never sign for PRs.
+  - buildConfig: Release
+  - buildConfig: Debug
 
-      - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-        - job: OSX
-          pool:
-            vmImage: 'macOS-latest'
-          strategy:
-            matrix:
-              debug_configuration:
-                _BuildConfig: Debug
-                _SignType: none
-              release_configuration:
-                _BuildConfig: Release
-                _SignType: none
-          steps:
-          - ${{ if ne(variables['System.TeamProject'], 'public') }}:
-            - task: Bash@3
-              displayName: Setup Private Feeds Credentials
-              inputs:
-                filePath: $(Build.SourcesDirectory)/eng/common/SetupNugetSources.sh
-                arguments: $(Build.SourcesDirectory)/NuGet.config $Token
-              env:
-                Token: $(dn-bot-dnceng-artifact-feeds-rw)
-          - script: eng/common/cibuild.sh
-              --configuration $(_BuildConfig)
-              --prepareMachine
-              --integrationTest
-              $(_InternalRuntimeDownloadArgs)
-            name: Build
-            displayName: Build
+extends:
+  ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
+    template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  ${{ else }}:
+    template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: $(DncEngInternalBuildPool)
+        image: 1es-windows-2022
+        os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
```

Comment:
We don't need this. It is tracking information for using the conversion tool, which we didn't use here.